### ISSUE TITLE
trying to get onClick() to stop after winCheck() executes.  

### ISSUE THREAD

**Issue by RpRoderick** [ga-wdi-boston/game-project#371]
```js
const onTileClick = function () {
  let tile = $(this).attr('class');
  let i = +(tile.replace(/\D/g, ''));
   if (world.ttt.player === 'x') {
     $('.'+tile).append('X');
     world.ttt.player = 'o';
     world.ttt.board[i] = 'x';
  } else if (world.ttt.player === 'o') {
     $('.'+tile).append('O');
     world.ttt.player = 'x'; 
     world.ttt.board[i] = 'o';
  }
    if (winCheck()) {
      $('table').css('pointer-events', 'none');  
 console.log('you win');
    }
};
```

debugger above win check is fine, debugger below if winCheck doesn't kick in.  this console.log above ('you win') doesn't work, but the console.log in the winCheck function itself does work and announces ('win') when you x or o a correct [i] sequence.  So, the if statement isn't kicking in here, I believe.   I have no linter errors at all.  no console errors pop up.  seems like a flow problem, but I can't see it.

**Comment by raq929** [comment]
Sorry, can you but those backtics on a separate line so I can read the text of your comment?

**Comment by RpRoderick** [comment]
awww.  oops.  I'll rewrite

**Comment by RpRoderick** [comment]
debugger above win check is fine, debugger below if winCheck doesn't kick in.  this console log ('you win') doesn't kick in, but the console.log in winCheck itself ('win') kicks in when you correctly hit x or o winning arrays.  So, I'd say that it's not hitting if statement of winCheck.  I can't see why.

**Comment by RpRoderick** [comment]
This should turn clicks off when winCheck () kicks in, but it does not.  wondering if it is because I have no return statement on my winCheck () itself?? would I set return to a boolean?

**Comment by RpRoderick** [comment]
AHH!  Pause on this please.  Something in my logic is screwy.  didn't notice before but winCheck is kicking in on some incorrect wins, not all, but some.  that may be it, I'll get back to you

**Comment by neugierige** [comment]
I don't see a `debugger` in the code you pasted...?

**Comment by raq929** [comment]
Quack Quack.

**Comment by RpRoderick** [comment]
I took out debuggers in posted code to unclutter.

**Comment by RpRoderick** [comment]
But, wincheck function does keep logging ('win') after win triggers, but that won't matter if i shut down clicks after winCheck goes through (because you won't be able to click and log win, because the game will be over). so back to original issue, how do I get if winCheck statement to kick in?

**Comment by RpRoderick** [comment]
hmm, Rachel had me switch to .off instead of .css, which makes sense, but the debugger still never gets there.

**Comment by neugierige** [comment]
when you say "there," I assume you mean inside the `if (winCheck())` block? 

What is `winCheck()`? What does it return?

**Comment by RpRoderick** [comment]
winCheck is the () which checks against the [i] click events to see if you have a win, if not you continue playing.  The ruturn is what I am wondering about as far as why it isn't working, it has no return.

**Comment by RpRoderick** [comment]
They are kicking us out of here.  I'll deploy in AM.

**Comment by neugierige** [comment]
that was my suspicion. remember that if-statements must evaluate a boolean expression in order to fire

**Comment by neugierige** [comment]
hey @RpRoderick are you still working on this one? can we close it yet?

**Comment by RpRoderick** [comment]
ok, so, solved.  I was requiring the global.js in events, and I was setting gameOver = board (which was inside of global.js file) however, because of truthy falsey (I believe), it wasn't working.  That means i had set x=y and then defined x=gameOver, which one would think means that y=gameOver because I had set x=y.  Not true.  JS didn't care for that.  So I simply bypassed the x=y step (which was redundant anyway) and set the global.js gameOver directly to my boolean.  It works.  And AHHHH!  Annoying.  Thank you Patrick for helping me figure it out.
